packages/calcite-components/src/components/panel/panel.stories.ts

```diff
@@ -440,3 +440,24 @@ export const withNoHeaderBorderBlockEnd_TestOnly = (): string =>
   html`<calcite-panel style="--calcite-panel-header-border-block-end:none;" height-scale="s" heading="My Panel"
     >Slotted content!</calcite-panel
   >`;
+
+export const contentTopSlot_TestOnly = (): string => html`
```

Comment:
New tests no longer need the `_TestOnly` suffixes. See updated [testing conventions](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/conventions/Testing.md#writing-stories).